packages/db/src/core/cli/commands/execute/index.ts

```diff
@@ -44,6 +44,7 @@ export async function cmd({
 		virtualModContents = getStudioVirtualModContents({
 			tables: dbConfig.tables ?? {},
 			appToken: appToken.token,
+			isBuild: false,
```

Comment:
Is there a reason for an `isBuild` parameter, when `execute` could set the app token in your environment to match how production builds work? That way, we could remove both the `appToken` and `isBuild` args from this function, and avoid maintaining two separate code paths.

Reply:
That would be nice, but does `execute` run in `dev` and `build` and all of the other commands?